force-app/main/default/lwc/actionBar/actionBar.js-meta.xml

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata" fqn="actionBar">
+    <!--current api version-->
```

Comment:
Why this comment?

Reply:
Made a dummy commit with the comment so that workflow is triggered and all the tests passed for the PR.

Reply:
I shall remove the comment and commit again if required.

Reply:
Ok, please remove. In the future you can just re-run CI from the [actions tab](https://github.com/trailheadapps/purealoe-lwc/actions).

Reply:
Done!